client/platform/desktop/backend/native/common.spec.ts

```diff
@@ -107,6 +109,25 @@ const console = new Console(process.stdout, process.stderr);
 
 const emptyCsvString = '# comment line\n# metadata,fps: 32,"whatever"\n#comment line';
 
+// Adding in some ingest pairs
+type testPairs = [string[], MultiTrackRecord, Record<string, Attribute>];
+
+const testData: testPairs[] = fs.readJSONSync('../testutils/viame.spec.json');
+const images: Record<string, string> = {};
+const imageList = Array.from(Array(10).keys());
+imageList.shift(); //remove 0.png
+// eslint-disable-next-line no-return-assign
+imageList.forEach((item) => images[`${item}.png`] = '');
+
+type TestKey = string | 'annotations.csv';
+const fileSystemData: Record<string, Record<string, string>> = { };
+testData.forEach((triplet, index) => {
+  fileSystemData[`test${index}`] = {
+    ...images,
+    'annotations.csv': triplet[0].join('\n'),
+  };
+});
```

Comment:
I did this because while we had the viame serializer test with the attributes that was passing.  We also had an invidiual test of attributes in `attributes.spec.ts` that was passing.  We didn't have a test that made sure these were hooked up together and working properly.   This is how we were getting pipelines that could run without having the attributes show up in the `meta.json`.  So this is running a fuller test from loading the images/annotations.csv from a mockfs directory so it ensures that viame serializer and the attributes processor are working together and producing the desired meta.json outcome.  Hopefully my newer comments cleared it up a bit.